ophyd/sim.py

```diff
@@ -139,6 +140,14 @@ def get(self):
         # example.
         return super().get()
 
+    def configure(self,
+                  d: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
+        '''Configure the device for something during a run
+        '''
+        old = self.read_configuration()
```

Comment:
Maybe add a check that the dictionary is empty (as there is nothing _to_ configure)?

Reply:
done